src/main/java/org/chtracker/BaseConfiguration.java

```diff
@@ -3,16 +3,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.InjectionPoint;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Scope;
 
+@EnableAutoConfiguration
 @Configuration
-public class LoggingConfiguration {
+public class BaseConfiguration {
```

Comment:
What is the reason behind naming class BaseConfiguration if it is atm only creating bean for logging. I think name might be misleading.

Reply:
it has @EnableAutoConfiguration also there will be other general purpose beans there.